### DISCUSSION TITLE
关于IC2中尝试使用<MFSU充电座>却无法在此上方为所需充电物品“站立式充电”问题

### DISCUSSION THREAD

**Discussion by WaterDouble** [IndustrialCraftMC/IndustrialCraft2#23] [【漏洞反馈】Bugs Report]
![image](https://user-images.githubusercontent.com/132697942/236632701-1dc9e698-c1ca-4f79-8ee5-2aa8dcd61d88.png)

使用1.19.2 forge-43.2.0 Op-I1-pre4

问题描述：不光是在<MFSU充电座>上，其他充电座均无法通过站立在充电座上进行有效充电，但将物品放置在充电座内即可正常充电。

**Comment by AbbottMc** [comment]
检查过了充电座里是有电的吗？

**Reply by WaterDouble** [reply]
旁边放置的是<创造模式发电机>，内部有电且充足。

**Reply by AbbottMc** [reply]
好的，之后我查查~
